### PR TITLE
Turn on cpp test to bypass abseil-py's build error

### PR DESCRIPTION
**Pull request by cynthiajoan**

### CODE REVIEW

**Comment by a-maurice** [comment]
I don't think that this is a great solution, because it'll mean building a ton of tests that we don't actually want.

**Comment by cynthiajoan** [comment]
> I don't think that this is a great solution, because it'll mean building a ton of tests that we don't actually want.

I agree that's not an ideal one, better solution is to use -DBUILD_TESTING=off, but unfortunately that doesn't work. And since we are not directly refer abseil-py, it's the libs like boringssl or curl did that, even the flag -DBUILD_TESTING=off is already set on cpp's CMakeList.txt, the error would still hit the second run.

**Review by chkuang-g — DISMISSED** [review]

**Comment by cynthiajoan** [comment]
Update the PR, only turn on stub flag for CPP. corresponding CPP PR is https://github.com/firebase/firebase-cpp-sdk/pull/528

File: build_bash.sh [hunk 1]
```diff
@@ -36,6 +36,7 @@ fi
 
 CMAKE_OPTIONS="${CMAKE_OPTIONS} -DUNITY_ROOT_DIR=${UNITY_ROOT_DIR}"
 CMAKE_OPTIONS="${CMAKE_OPTIONS} -DFIREBASE_UNITY_BUILD_TESTS=ON"
+CMAKE_OPTIONS="${CMAKE_OPTIONS} -DFIREBASE_CPP_BUILD_STUB_TESTS=ON"
```
Comment (a-maurice) [inline build_bash.sh:39]: Could you add a comment explaining why this is needed

Comment (cynthiajoan) [reply build_bash.sh]: DONE

**Review by a-maurice — APPROVED** [review]

**Pull request merged**